src/screens/Ticketing/Purchase/Travellers/index.tsx

```diff
@@ -35,14 +36,25 @@ const Travellers: React.FC<TravellersProps> = ({
 
   const {userProfilesWithCount, addCount, removeCount} = useUserCountState();
 
+  const {tariffZones} = useTicketState();
+  const {
```

Comment:
For now just using `tariffZones[0]` as the default, which conveniently is _Sone A_. We probably want a more reliable method for selecting the default zone. Should the default zone already be flagged when it is returned from backend?